tests/platform_helper/test_command_environment.py

```diff
@@ -700,7 +710,8 @@ def test_generate_terraform(
         }
 
         mocked_session = MagicMock()
-        mock_get_aws_session.return_value = mocked_session
+        mock_get_aws_session_1.return_value = mocked_session
+        mock_get_aws_session_2.return_value = mocked_session
```

Comment:
```suggestion
        mock_get_aws_session_1.return_value = mock_get_aws_session_2.return_value = mocked_session
```

not sure if this is more readable or not / what black would do with it